metrics/models.go

```diff
@@ -10,47 +10,27 @@ import (
 	base "github.com/w3c/wptdashboard/shared"
 )
 
-//
-// Sortable slices of test runs
-//
-type TestRunSlice []base.TestRun
-
-func (s TestRunSlice) Len() int {
-	return len(s)
-}
-
-func (s TestRunSlice) Less(i int, j int) bool {
-	if s[i].Revision < s[j].Revision {
-		return true
-	}
-	if s[i].Revision > s[j].Revision {
-		return false
-	}
-	if s[i].BrowserName < s[j].BrowserName {
-		return true
-	}
-	if s[i].BrowserName > s[j].BrowserName {
-		return false
+// ByCreatedDate sorts tests by their most-recent created date.
+type ByCreatedDate []base.TestRun
+
+func (s ByCreatedDate) Len() int          { return len(s) }
+func (s ByCreatedDate) Swap(i int, j int) { s[i], s[j] = s[j], s[i] }
+func (s ByCreatedDate) Less(i int, j int) bool {
```

Comment:
Maybe add comment that it is descending *Revision*, but ascending *everything else*.

Reply:
Done.